api/external/cfgmgmt/cfgmgmt.proto

```diff
@@ -130,7 +130,7 @@ service ConfigMgmt {
   };
 
   /*
-  GetMissingNodeDurationCounts
+  List Missing Nodes Count 
```

Comment:
Singular 'count' is correct

Reply:
But perhaps 'Show'? I can see it either way.

Reply:
I went with "List" because it will show a list of missing nodes over various durations, per the example:
> cfgmgmt/stats/missing_node_duration_counts?durations=3d&durations=1w&durations=2w&durations=1M&durations=3M